src/routes/auth/user/+layout.svelte

```diff
@@ -0,0 +1,34 @@
+<script>
+  import '../../../app.css';
+  import Main from './components/Main.svelte';
+  import Footer from './components/Footer.svelte';
+  import Header from './components/Header.svelte';
```

Comment:
I can't notize `Main` and `Footer` components, does `vite` allow to compile with unused elements?